tree/tree/src/TTree.cxx

```diff
@@ -1336,7 +1336,7 @@ Long64_t TTree::AutoSave(Option_t* option)
 {
    if (!fDirectory || fDirectory == gROOT || !fDirectory->IsWritable()) return 0;
    if (gDebug > 0) {
-      printf("AutoSave Tree:%s after %lld bytes written\n",GetName(),fTotBytes);
+      printf("AutoSave Tree:%s after %lld bytes written\n",GetName(),GetTotBytes());
```

Comment:
Please use the ROOT printing routine (Info is likely to be the one you are looking for